src/guide/testing-the-jdk.md

```diff
@@ -30,43 +30,53 @@ In-depth documentation about the jtreg framework is found here: [jtreg harness](
 
 Below is a small example of a jtreg test. It’s a clean Java class with a main method that is called from the test harness. If the test fails we throw a RuntimeException. This is picked up by the harness and is reported as a test failure. Try to always write a meaningful message in the exception. One that actually helps with understanding what went wrong once the test fails.
 
-    /*
-     * @test
-     * @summary Make sure feature X handles Y correctly
-     * @run main TestXY
-     */
-    public class TestXY {
-        public static void main(String[] args) throws Exception {
-            var result = X.y();
-            if (result != expected_result) {
-                throw new RuntimeException("X.y() gave " + result + ", expected " + expected_result);
-            }
+~~~Java
+/*
+ * @test
+ * @summary Make sure feature X handles Y correctly
+ * @run main TestXY
+ */
+public class TestXY {
+    public static void main(String[] args) throws Exception {
+        var result = X.y();
+        if (result != expected_result) {
+            throw new RuntimeException("X.y() gave " + result + ", expected " + expected_result);
         }
     }
+}
+~~~
 
 This example only utilizes three jtreg specific tags, `@test`, `@summary`, and `@run`. `@test` simply tells jtreg that this class is a test, and `@summary` provides a description of the test. `@run` tells jtreg how to execute the test. In this case we simply tell jtreg to execute the main method of the class `TestXY`. `@run` isn't strictly necessary for jtreg to execute the test, an implicit `@run` tag will be added if none exists. However, for clarity and in order to avoid bugs it's recommended to always explicitly use the `@run` tag.
 
 There are several other tags that can be used in jtreg tests. You can for instance associate the test with a specific bug that this test is a regression test for.
 
-    @bug 7000001
+~~~
+@bug 7000001
+~~~
 
 Or you can specify a number of requirements that must be fulfilled for jtreg to execute the test.
 
-    @requires docker.support
-    @requires os.family != ”windows”
-    @requires os.maxMemory > 3G
-    @requires os.arch=="x86_64" | os.arch=="amd64"
+~~~
+@requires docker.support
+@requires os.family != ”windows”
```

Comment:
```suggestion
@requires os.family != "windows"
```
The quotes should be plain for easier copy-and-paste.

Reply:
Good catch!